### DISCUSSION TITLE
How to create an image with blurry loading effect in NextJS | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#66] [Announcements]
# How to create an image with blurry loading effect in NextJS | Leo's dev blog – stories, insights, and ideas

NextJS has provided a built-in image component that has many useful features, we can leverage them with some custom styles to create a beautiful image with a blurry loading effect.

https://www.leohuynh.dev/blog/how-to-create-an-image-with-blurry-loading-effect-in-nextjs

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
